wiki/Contests/TKBC/1/en.md

```diff
@@ -0,0 +1,82 @@
+---
+tags:
+- TKBC
+- TKBC1
+---
+
+# Taiko Korean Beatmap Contest #1
+
+![TKBC1 Logo](img/logo.png)
+
+The **Taiko Korean Beatmap Contest #1** (***TKBC1***) is a osu!taiko mapping contest for only Koreans. It is the first installment of the TKBC.
+
+## Contest schedule
+
+| Event | Timestamp (UTC +9) |
+| --: | :-- |
+| Registration Phase | 2020-04-21/2020-04-26 |
+| Submitting Phase | 2020-04-27/2020-05-10 |
+| Judging Phase | 2020-05-11/2020-05-31 |
+| Results Announcement | 2020-06-01 |
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
+- The difficulty should be Oni or higher.
+- If you submit several maps, the last one would be your entry.
+- You should not reveal your map before the results announcing.
+- You can't submit what you uploaded already.
+- Only Koreans can participate this contest.
+- Judges can't participate this contest.
+
+### Metadata rules
+
+- Artist should be asdf.
+- Title should be 1223.
+- Difficulty should be Oni or Inner Oni.
+- Source and Tags should be blank.
+- When you submit your map, contain the metadata to the mail.
```

Comment:
```suggestion
- Artist must be `asdf`.
- Title must be `1223`.
- Difficulty must be Oni or Inner Oni.
- Source and tags must be left blank.
- When submitting your map, include the metadata in the email.
```
If these are rules, then I think "should" should be replaced with "must". Should is suggestive/recommended while must is required.

The last rule seems to be worded weirdly, I tried fixing it, but I'm not sure if it is saying the same thing?